package/libs/libnetfilter-queue/Makefile

```diff
@@ -9,13 +9,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnetfilter_queue
 PKG_VERSION:=1.0.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:= \
-	http://www.netfilter.org/projects/libnetfilter_queue/files/ \
-	ftp://ftp.netfilter.org/pub/libnetfilter_queue/
-PKG_MD5SUM:=df09befac35cb215865b39a36c96a3fa
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=git://git.netfilter.org/libnetfilter_queue
```

Comment:
Ahh, apparently you need a pretty new certificate repo for let's encrypt domains...
Oh well, http it is then.... still preferred over git.